packages/client/src/worker/protocol.ts

```
@@ -152,6 +152,32 @@ export interface SharingDescriptor {
   grants: SharingGrantDescriptor[] | null;
 }
 
+/**
+ * The engine's verdict on a received share's latest resolve (mirrors
+ * `ResolutionClass`). Only the engine reaches one — a host renders it.
+ */
+export type ReceivedShareResolution =
+  | 'granted'
+  | 'revocation-signal'
+  | 'unresolvable'
+  | 'epoch-lag';
```

ESLint

[error] Replace `|·'granted'⏎··|·'revocation-signal'⏎··|·'unresolvable'⏎·` with `'granted'·|·'revocation-signal'·|·'unresolvable'`

(prettier/prettier)